scripts/net/run-sample-tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This was so minimal change that I did not create separate commit for it.

But if using minimal `/bin/sh` the script does not work. It seem to rely on some Bash specific `source SCRIPT` instead of `. SCRIPT`